docs/core_docs/docusaurus.config.js

```diff
@@ -197,6 +197,21 @@ const config = {
               },
             ],
           },
+          {
+            type: "dropdown",
+            label: "v0.2",
+            position: "right",
+            items: [
+              {
+                label: "v0.2",
+                href: "/docs/introduction",
+              },
+              {
+                label: "v0.1",
+                href: "/../v0.1/docs/get_started/introduction",
```

Comment:
this seems to not work in the preview. Why are you going up a level?

Reply:
```suggestion
                href: "https://js.langchain.com/v0.1/docs/get_started/introduction",
```
baseurl stuff